src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -49,6 +49,28 @@ def can_edit_alert_rule(rule, organization, user_id, user):
     return True
 
 
+def get_rule(data, request, rule_id):
+    if data.get("rule") and data.get("alert_rule"):
+        raise serializers.ValidationError("Pass either rule or alert rule, not both.")
```

Comment:
@ceorourke Maybe we could have `rule_type` and `rule_id` as the arguments instead so we don't have the possibility of conflicts?

Reply:
Yeah I could see that, you mean for the post method too right?

Reply:
@ceorourke yea should probably be aligned. If it's too much work to refactor then not worth it.

Reply:
Ok pushed that refactor up